app/assets/stylesheets/comments.scss

```diff
@@ -322,33 +347,7 @@ a.header-link {
         @extend %kbd;
       }
     }
-    .actions {
```

Comment:
I couldn't see when this CSS is used. Maybe it's still required but for the comment forms, there is `.actions` defined with more specificity up top.